File: java/assign4/src/Banner.java
package BannerPrice;

public class Banner{

	private int height;
	private int width;


//	public Banner(){
//	
//		this(10,12);
//	}

	public Banner(int h, int w){
		
		height = h;
		width = w;
	}
	
	public int getHeight(){
		
		return height;
	}
	public void setHeight(int h){
	
		height = h;
	}

	public int getWidth(){
	
		return width; 
	} 

	public void setWidth(int w){
	
		width = w;
	}
	public double area(){
		
		return height * width ;
	
	}
}

File: dotnet/assignment3/DemoApp/Banner.cs
namespace bannerPrice;

public class Banner{

    private int height;
    private int width;

    public Banner(int h , int w){

        height = h;
        width = w;
    }

    public int Height{
        get { return height;}
        set {height = value;}
    }

    public int Width{
        get {return width;}
        set { width = value;}
    }
    public virtual double area(){
        return height * width;
    }







}



